src/_component/welcome/welcome.pug

```diff
@@ -24,8 +24,18 @@ block awesome-content
     .header-wrapper-orange
       .header-orange= info
 
-    .box__info-box
-      .text__section-title= moreTitle
+    .flex-gap16
+      .box__gap-8
```

Comment:
? why do you need this if there are no children in it?

Reply:
?

Reply:
let me check